lib/ip.go

```diff
@@ -15,6 +16,18 @@ type IP uint32
 // IPStr is a string representation of an IPv4 address.
 type IPStr string
 
+// IP4Range is a numerical representation of an IPv4 range.
+type IP4Range struct {
+	startIP IP
+	endIP   IP
+}
+
+// IP6Range is a numerical representation of an IPv4 range.
+type IP6RangeInt struct {
```

Comment:
No need to call this `IP6RangeInt`; we already call `IP4Range` without the ending `Int` in the name, even though it stores integer forms of IPs. So just call it `IP6Range`.

Reply:
The same issue with this as well, we already have an `IP6Range` in the `ip6_range.go` file and that type have  two uint32 blocks and I need big int so I can't use that type

Reply:
Alright so let's leave the name the same here then.